auxil/spicy/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Spicy needs the full prefix flex and bison while Zeek captures only the paths
+# to the executables. Derive the prefixes from the binary paths if given under
+# the assumption that their bindir is under their prefix (which also implies
+# that one such prefix even exists).
```

Comment:
Could this move either into Spicy's cmake, or into Zeek higher-level CMake? I think it would be preferable to avoid this additional layer of CMake tweaking the Spicy build here, and then just have `auxil/spicy` be the submodule directly, just to remove complexity and follow the layout/structure for other submodules.